inc0gnito-ctf-2015/.gitignore

```diff
@@ -0,0 +1,3 @@
+network/Hack-a-network-administrator's-terminal/network_hacking_102.zip
+forensic/POS/POS_9b648dbca2017349512c89466391c07d722418d1
+forensic/old_school/old_school_65c87efde678662c9139c81e016a62671cea2d6f
```

Comment:
hey did you add these files some where in the cloud ? 

because I couldn't see the links of them added

if you have the files with you email me and I will add them here :+1: